### ISSUE TITLE
ValueError: Unable to find google arts image token

### ISSUE THREAD

**Issue by kolt54321** [expenses/GAPDownloader#22]
I'm getting the above error on all images that I try (example [here](https://artsandculture.google.com/asset/hanging-scroll-lu-zhi/hwGFyXp0lmP7qg)). @lovasoa Is there any chance you are able to fix the token extractor?

**Comment by lovasoa** [comment]
Hello! Google slightly changed the format of the pages. Dezoomify-rs was updated, you should be able to download your images with it.

**Comment by kolt54321** [comment]
Hi! Thanks for the update. Unfortunately I've built quite a bit of code around batch downloading and renaming per author here (with some updates I didn't previously have a chance to push - added now). Is it much trouble to bring over the changes here as well? This is the only option I'm aware of that makes batch-downloading easy and organizes the outputs.

**Comment by lovasoa** [comment]
No, it shouldn't be hard at all to port the changes, it's just a regex to update. PR welcome!

**Comment by kolt54321** [comment]
Thanks so much! Very much appreciated :)

**Comment by Emasoft** [comment]
Any update on this?

**Comment by Explorare** [comment]
> Any update on this?

Have you tried this project?
[lovasoa/dezoomify: Dezoomify is a web application to download zoomable images from museum websites, image galleries, and map viewers. Many different zoomable image technologies are supported.](https://github.com/lovasoa/dezoomify)

**Comment by Emasoft** [comment]
@Explorare Yes, but I'm using this code in a python project. I tried to use their RegEx string anyway, but I get a '400: Bad Request' after a while.
